packages/perseus-editor/src/__tests__/items-extras-editors.test.tsx

```diff
@@ -0,0 +1,59 @@
+import "@testing-library/jest-dom";
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+
+import ItemExtrasEditor from "../item-extras-editor";
+
+describe("ItemExtrasEditor", () => {
+    it("should render correctly with default props", () => {
```

Comment:
**thought**: I wonder if we should assert all of the "extras" here given the test name.  Otherwise I'd be inclined to name this `'should render period table option as unchecked with default props'`.